configs/datasets/classification_last_one_lvl3MC.yml

```diff
@@ -0,0 +1,43 @@
+path: /groups/icecube/petersen/GraphNetDatabaseRepository/Leon2022_DataAndMC_CSVandDB_StoppedMuons/last_one_lvl3MC.db
```

Comment:
If you add this, and the ditto model config file, I figure you should use these in your example script. Otherwise I would just keep these in the relevant analysis repo.